nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/home/layout1.html

```diff
@@ -3,53 +3,54 @@
         <div class="container">
             <div class="row row1 module">
                 <div class="span12">
-                    <p>Hei {{ c.userobj.display_name }}!</p>
-                    <b>Palvelukatalogi</b>-sivulla voit tarkastella ja hakea palvelussa julkaistuja
-                    rajapintoja.<br />
-                    <b>Organisaatiot</b>-sivulla on listattuna palveluun rekisteröityneet organisaatiot.
+                    <p>{{ _('Hei') }} {{ c.userobj.display_name }}!</p>
+                    <b>{{ _('Datasets') }}</b>-{% trans %}sivulla voit tarkastella ja hakea palvelussa julkaistuja
```

Comment:
So these texts can't be used from translations yet? Like -> _('sivulla voit tarkastella ja...')

Reply:
Yes we can, but we shouldn't yet.
_('Text string') and {% trans %} Text string { % transend %} macros provide the same functionality. Trans-tags are mainly for larger text blocks and _('...') is for shorter text.
I have wrapped all our custom strings in ckan templates with translation macros.

The point, that I tried to clarify above, was that we should use english sentences and words in our ckan templates, so we dont get strange mixed language translation files when we eventually generate them. English should be the base language for a CKAN plugin. 
This is not a blocker for this pull request, as this mainly concerns template style changes. The text string fixes will be a separate ticket.